### PR TITLE
AG-12903 v31 Deprecations Removed

### PR DESCRIPTION
**Pull request by StephenCooper**
https://ag-grid.atlassian.net/browse/AG-12903

### CODE REVIEW

File: packages/ag-grid-community/src/filter/provided/date/dateCompWrapper.ts [hunk 1]
```diff
@@ -107,19 +107,9 @@ export class DateCompWrapper {
     }
 
     public updateParams(params: IDateParams): void {
-        let hasRefreshed = false;
         if (this.dateComp?.refresh && typeof this.dateComp.refresh === 'function') {
-            const result = this.dateComp.refresh(params);
+            this.dateComp.refresh(params);
             // framework wrapper always implements optional methods, but returns null if no underlying method
```
Comment (peterjrreynolds) [inline packages/ag-grid-community/src/filter/provided/date/dateCompWrapper.ts]: Can remove this comment too

File: packages/ag-grid-enterprise/src/multiFilter/multiFloatingFilter.ts [hunk 2]
```diff
@@ -91,17 +87,8 @@ export class MultiFloatingFilterComp extends Component implements IFloatingFilte
...
-                    // framework wrapper always implements optional methods, but returns null if no underlying method
-                    if (result !== null) {
-                        hasRefreshed = true;
-                    }
-                }
-                if (!hasRefreshed) {
-                    floatingFilter.onParamsUpdated?.(floatingFilterParams);
-                }
+                floatingFilter.refresh?.(floatingFilterParams);
+                // framework wrapper always implements optional methods, but returns null if no underlying method
```
Comment (peterjrreynolds) [inline packages/ag-grid-enterprise/src/multiFilter/multiFloatingFilter.ts]: Can remove comment

File: packages/ag-grid-react/src/shared/customComp/filterComponentWrapper.ts [hunk 3]
```diff
@@ -73,7 +73,7 @@ export class FilterComponentWrapper
         // remove props in IFilterParams but not CustomFilterProps
         delete (props as any).filterChangedCallback;
         delete (props as any).filterModifiedCallback;
-        delete (props as any).valueGetter;
+        delete (props as any).getValue;
```
Comment (peterjrreynolds) [inline packages/ag-grid-react/src/shared/customComp/filterComponentWrapper.ts]: This shouldn't be here - `getValue` is in `CustomFilterProps`

File: packages/ag-grid-angular/projects/ag-grid-angular/src/lib/ag-grid-angular.component.ts [hunk 4]
```diff
@@ -1218,12 +1173,6 @@ export class AgGridAngular<TData = any, TColDef extends ColDef<TData> = ColDef<a
      * @initial
      */
     @Input() public serverSideInitialRowCount: number | undefined = undefined;
-    /** When `true`, the Server-side Row Model will suppress Infinite Scrolling and load all the data at the current level.
-     * @default false
-     * @initial
-     * @deprecated v31.1
-     */
-    @Input() public suppressServerSideInfiniteScroll: boolean | undefined = undefined;
```
Comment (StephenCooper) [inline packages/ag-grid-angular/projects/ag-grid-angular/src/lib/ag-grid-angular.component.ts:1226]: Need confirmation on this property first.

Comment (StephenCooper) [reply packages/ag-grid-angular/projects/ag-grid-angular/src/lib/ag-grid-angular.component.ts:1226]: Ok to remove this now.

**Review by peterjrreynolds — APPROVED** [review]

**Pull request merged**